pom.xml

```diff
@@ -249,7 +249,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <proto-backwards-compatibility.version>1.0.7</proto-backwards-compatibility.version>
 
     <swagger-annotations-version>1.5.4</swagger-annotations-version>
-    <snakeyaml.version>1.26</snakeyaml.version>
+    <snakeyaml.version>1.32</snakeyaml.version>
```

Comment:
looks like 1.33 is out https://bitbucket.org/snakeyaml/snakeyaml/wiki/Changes

Reply:
Thanks, but 1.32 is enough to fix reported CVE.